content/languages/riscv/cgreen.md

```diff
@@ -0,0 +1,43 @@
+---
+tags:
+  - riscv
+  - reference
+  - testing
+---
+
+# Cgreen
+
+Unit tests for RISC-V are written in C with [Cgreen](https://cgreen-devs.github.io/cgreen/) while the solution file is in assembly.
+
+## Basic Setup
+
+### Solution
+
+```asm
```

Comment:
I think `riscv` is supported by shiki. https://github.com/shikijs/shiki/blob/main/docs/languages.md#all-languages

```suggestion
```riscv
```